doc: Refactor "install boundary" section (#4557)
hashicorp/boundary: website/content/docs/install-boundary/configure-controllers.mdx

Before you can start Boundary, you must initialize the database from one Boundary controller.
This operation is only required once; it will execute the required database migrations for the Boundary cluster to operate.

```shell-session
boundary database init -config /etc/boundary.d/controller.hcl
```

> Boundary will automatically generate a number of resources to make getting started easier. Default scopes, auth methods,
> user, account, and targets are just some of the resources Boundary will generate unless you tell it not to. These
> resources, however, are not required. You can choose to skip the creation of these initial resources by adding some
> flags.

Reviewer: ```suggestion
Boundary automatically generates a number of resources to make getting started easier. Default scopes, auth methods,
user, account, and targets are some of the resources Boundary automatically generates unless you configure it not to. These
resources, however, are not required. You can add the following flags to skip creating these initial resources:
```

   -config /etc/boundary.d/controller.hcl
```

> For additional options, view the help documentation.

Reviewer: ```suggestion
Use the following command to view the help for additional options:
```

Upon logging in to Boundary for the first time, HashiCorp recommends creating an admin user for the global and project level scopes to manage Boundary.
This allows you to configure targets within those scopes and manage them.

> We recommend that you use the [KMS recovery workflow](/boundary/docs/install-boundary/initialize#log-in-with-recovery-kms) to log in to Boundary for the first time.

Reviewer: ```suggestion
HashiCorp recommends that you use the [KMS recovery workflow](/boundary/docs/install-boundary/initialize#log-in-with-recovery-kms) to log in to Boundary for the first time.
```
